xbmc/cores/dvdplayer/DVDPlayer.cpp

```diff
@@ -835,7 +835,7 @@ bool CDVDPlayer::OpenDemuxStream()
   int64_t len = m_pInputStream->GetLength();
   int64_t tim = m_pDemuxer->GetStreamLength();
   if(len > 0 && tim > 0)
-    m_pInputStream->SetReadRate((unsigned int) (g_advancedSettings.m_readBufferFactor * len * 1000 / tim));
```

Comment:
FYI: The variable set by calling SetReadRate() is the same as m_writeRate used in CFileCache below. See https://github.com/arnova/xbmc/blob/cache_full_debug/xbmc/filesystem/FileCache.cpp#L544 . Or am I misunderstanding you?